emcc.py

```diff
@@ -859,10 +862,8 @@ def detect_fixed_language_mode(args):
                 break
             libs.append((i, l))
             newargs[i] = ''
-          elif 'WASM_OBJECT_FILES=1' in settings_changes and shared.Building.is_wasm(arg):
-            input_files.append((i, arg))
           else:
-            logger.warning(arg + ' is not valid LLVM bitcode')
+            logger.warning(arg + ' is not a valid input')
```

Comment:
nit: "not a valid input file" maybe? makes it slightly clearer to disambiguate the file's contents from the filename

Reply:
Fixing, thanks.